components/centraldashboard/screen-diff-testing/diff-test.js

```diff
@@ -0,0 +1,43 @@
+import puppeteer from 'puppeteer'
```

Comment:
Even though this is test code, we should probably make sure it fits with the eslint style we've applied to our other components. I think we need to add the `ecmaVersion` key to the `parserOptions` property in the the `.eslintrc.json` file at the root of the folder.

Reply:
Done!